### PR TITLE
DeclareStrictTypesFixer - fix for "strict_types" set to "0"

### PR DESCRIPTION
**Pull request by kubawerlos**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4687

Ping @Seldaek, @SpacePossum (as authors) and @JeroenVanOort (as bug reporter) for review.

### CODE REVIEW

File: src/Fixer/Strict/DeclareStrictTypesFixer.php [hunk 1]
```diff
@@ -82,62 +82,48 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
...
             return;
         }
 
-        $this->fixStrictTypesCasing($tokens, $sequenceLocation);
-    }
-
-    /**
-     * @return Token[]
-     */
-    private function getDeclareStrictTypeSequence()
```
Comment (GrahamCampbell) [inline src/Fixer/Strict/DeclareStrictTypesFixer.php]: Why is this removed. The new code is slower and harder to read now, since the line is so long?

Comment (kubawerlos) [reply src/Fixer/Strict/DeclareStrictTypesFixer.php]: @GrahamCampbell this is removed because the new code is faster (yes, it is) and easier to read now - I'd say one 152 characters long line is more readable than 27 lines long function.

Comment (julienfalque) [reply src/Fixer/Strict/DeclareStrictTypesFixer.php]: I agree with @kubawerlos. Also static variables inside functions are easy memory leaks.

Comment (kubawerlos) [reply src/Fixer/Strict/DeclareStrictTypesFixer.php]: To clarify:
- it is faster in runs that are not changing the code - for `findSequence` 6 new objects were created every time, and now none are created
- `getDeclareStrictTypeSequence` had to be updated anyway, as the check for `declare` could not use objects, because we have to search for any number, not only `1` - which is the core of this bugfix

**Comment by julienfalque** [comment]
Thank you @kubawerlos.
